### ISSUE TITLE
Option to get percentages in barplots of descriptives

### ISSUE THREAD

**Issue by victor-moreno** [jamovi/jmv#286]
Currently only descriptives provides a barplot but only counts.
I think an option to plot percentages using the total of the splitby variable would be useful to make graphical comparisons. 

contTables would be al alternative location for these plots. There, percentages could be calculated according to rows or columns based on te selection. 

I can code this if is considered a worth addition.

**Comment by jonathon-love** [comment]
i've been agitating for this for a while :) @raviselker ?

**Comment by raviselker** [comment]
Yeah, the problem is that at some point we also allowed for bar plots for continuous variables and percentages don't make sense for these plots.

**Comment by victor-moreno** [comment]
Agree that barplots with percentages only make sense for categorical variables and probably they should be under a new "Plots" menu for contingency tables. The one in descriptives can be used as template.
